### PR TITLE
[Blockchain] An Overview of web 3.O

### LINKED ISSUE #4645: [Blockchain] An Overview of Web 3.0
#### Pre-submission advice
By following all our pre-submission advice and reviewing our Resources folder, you will maximise your chances of your topic being approved.

We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner.

Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles that will help you propose and write a successful article.

**Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

#### Proposed title of article
An Overview of Web 3.0 

#### Proposed article introduction
Web 3.O is an internet powered by Blockchains. It is the internet that was not possible before all the benefits that Blockchain provides. These benefits include the internet of value, decentralization, trust minimization, secure digital identity, and more. All these characteristics help Web 3.O create new business models and reinvent how the internet works.

When people talk of Web 3.O, they often ask how the future internet will be different from what we have today and what we have had in the past.

The emergence of Blockchain means we will be able to consciously build Web 3.O in a way that works for everyone. This article will outline how the web has evolved, where it's going, and why we need Web 3.O.

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.
1. The Evolution of the Web
- What is Web 1.O
- What is Web 2.O
- What is Web 3.O
2. Why we need Web 3.0
3. How Blockchain works
4. What the Web 3.0 Stack looks like

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by JudyNduati**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?
This closes #4645

### CODE REVIEW

**Comment by JudyNduati** [comment]
@ahmadmardeni1, I opened a PR

**Comment by JudyNduati** [comment]
This closes #4645

**Comment by srishilesh** [comment]
@JudyNduati Could you please add the PR description back? And, ensure you have followed all the checks. (Also, please check the boxes, if done)

File: content/articles/an-overview-of-web-3-0/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+The internet just got a lot more complicated. There's an increased awareness of web 3, and there is so much jargon in the space, and it isn't easy to understand what it's all about. This article will break down all the jargon for you and make you perfectly understand what web 3.O is all about.
+
+This article will explain Web 3 or the future of the web. If you want to stay up-to-date on how we'll change our lives soon, make sure you read this article to get the complete picture. Without further ado, let's get started.
+
+### Table of Contents
+[The Evolution of the Web](#the-evolution-of-the-web)
+[Why we need Web 3](#why-we-need-web-3)
+[How blockchain works](#how-blockchain-works)
+[Blockchain to make Web 3 possible](#blockchain-to-make-web-3-possible)
+
```
Comment (jumamiller) [inline content/articles/an-overview-of-web-3-0/index.md:14]: Fix the table of contents, each with its own line.

File: content/articles/an-overview-of-web-3-0/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+The World wide web is a crucial element of our modern life. It has become part of us, and it would not be easy to outgrow our lives without it. This is because technology has greatly advanced as people can efficiently work remotely and easily communicate with the help of the internet. The reliance on the world wide web has grown in the last three decades and has gone through stages of evolution from [Web 1.O](https://websitebuilders.com/how-to/glossary/web1/), [Web 2.O](https://en.wikipedia.org/wiki/Web_2.0), and [Web 3.O](https://web3.foundation/).
```
Comment (jumamiller) [inline content/articles/an-overview-of-web-3-0/index.md:1]: World Wide Web and not World wide web, also consider adding (WWW) which much popular.

File: content/articles/an-overview-of-web-3-0/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+Let's dive deeper to find out how the internet evolved from its initiation to what we know it as today.
+
+#### What is Web 1.O
+Web 1.O was the first phase of the web revolution, also referred to as read-only web. It is primarily categorized by static websites that don't do much but display information on the page. You can read text, click different links, and write and send emails but can't attach or upload images and documents.
+
+Web 1.O was a content delivery network (CDN) that showcased information on websites. People mainly used it in the year 1990 to 2005. It had a small number of creators but a vast amount of consumers.
+
+#### What is Web 2.O
+Web 2.O is the interactive version of 1.0 since you can read and write on the internet as well. There are many content creators in Web 2.O since we have Youtube, Google, Microsoft, and social media applications.
```
Comment (jumamiller) [inline content/articles/an-overview-of-web-3-0/index.md:30]: Rewrite this sentence, using 'as well' makes it hard to follow.

File: content/articles/an-overview-of-web-3-0/index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+The internet and web technology have certainly evolved in the past few years. The world wide web has evolved through different phases: Web 1.O, Web 2.O, and Web 3.O. Each phase has brought about new technologies, tools, and techniques relevant to recruiters.
+
+![Evolution of web](/engineering-education/an-overview-of-web-3-o/web-evolution.jpg)
+
+[Image source](https://martech.zone/web-1-web-2-web-3-0/)
+
+Let's dive deeper to find out how the internet evolved from its initiation to what we know it as today.
+
+#### What is Web 1.O
+Web 1.O was the first phase of the web revolution, also referred to as read-only web. It is primarily categorized by static websites that don't do much but display information on the page. You can read text, click different links, and write and send emails but can't attach or upload images and documents.
```
Comment (jumamiller) [inline content/articles/an-overview-of-web-3-0/index.md:25]: You should be using a transition on this paragraph, i'e,   "For example,you can read text" to something like. Keep your sentences as simple as possible.

File: content/articles/an-overview-of-web-3-0/index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
+The World wide web is a crucial element of our modern life. It has become part of us, and it would not be easy to outgrow our lives without it. This is because technology has greatly advanced as people can efficiently work remotely and easily communicate with the help of the internet. The reliance on the world wide web has grown in the last three decades and has gone through stages of evolution from [Web 1.O](https://websitebuilders.com/how-to/glossary/web1/), [Web 2.O](https://en.wikipedia.org/wiki/Web_2.0), and [Web 3.O](https://web3.foundation/).
+
+Most people are probably wondering what the future web will be like. Well, Web 3.O (Web 3) is the future web. Many people are confused about what the next generation internet (Web 3) will be like. Web 3 is a part of a much larger phase of transition that we will go through. Therefore, there is a more significant change going on right now on the internet, and lots of improvements are being deployed.
```
Comment (jumamiller) [inline content/articles/an-overview-of-web-3-0/index.md:3]: Using preposition at the end of your sentence? It's less formal though? could you check it out?

**Review by jumamiller — CHANGES_REQUESTED** [review]
Hello @JudyNduati, minor edits and we're good!

**Comment by WanjaMIKE** [comment]
@JudyNduati could you paraphrase this portion it seems to show some plagiarism `To let it free and make its own decisions, we'll need a decentralized service that no one controls. Combining this capability with natural language processing (NLP), computers can easily understand information like humans, thus providing faster and more relevant results. They become more intelligent to satisfy people's needs.`

**Comment by JudyNduati** [comment]
Hello @jumamiller , @WanjaMIKE I made changes

**Review by jumamiller — APPROVED** [review]
@JudyNduati, great article!

**Comment by hectorkambow** [comment]
Thank you for your patience as our team has been working diligently to work through the recent increased volume of articles coming in. 

We are going through are ready articles in the order they were received and working on improving our review time as well. 

🚀

**Comment by hectorkambow** [comment]
jumping on this one. 🔍
